frontend/src/pages/accounts/[id].jsx

```diff
@@ -55,13 +55,11 @@ class AccountDetail extends Component {
           }
           border={false}
         >
-          {accountFetchingError?.status === 404 ? (
+          {accountFetchingError ? (
             <Translate
               id="page.accounts.error.account_not_found"
-              data={{ account_id: accountFetchingError.account_id }}
+              data={{ account_id: account.accountId }}
             />
-          ) : accountFetchingError ? (
-            <Translate id="page.accounts.error.account_fetching" />
```

Comment:
not-found and failure to receive a response from the server should be distinguishable as otherwise, Explorer will misreport existing users as non-existing when in fact it was just the broken backend response.